tensorflow/core/ops/mkl_nn_ops.cc

```diff
@@ -1994,6 +1976,54 @@ Uses oneDNN APIs to perform fused batch normalization and relu.
 expected to invoke these operators.
 )doc");
 
+REGISTER_OP("_MklNativeQuantizedConv2D")
+    .Input("input: Tinput")
+    .Input("filter: Tfilter")
+    .Input("min_input: float")
+    .Input("max_input: float")
+    .Input("min_filter: float")
+    .Input("max_filter: float")
+    .Output("output: out_type")
+    .Output("min_output: float")
+    .Output("max_output: float")
+    .Attr("Tinput: quantizedtype")
+    .Attr("Tfilter: quantizedtype")
+    .Attr("T: quantizedtype")
```

Comment:
Is `T` needed? Same for other native ops in this PR.

Reply:
It is not needed for this particular op as this `T` is needed for Conv only in non-native path to enable MklToTF conversion. Required datatypes for Conv are named differently (Tfilter, Tbias). So removed it. However, in other ops like Max/AvgPooling, `T` is the required datatype for input/output. Thank you for pointing out this!

Reply:
Thank you for the explanation!